src/specklepy/objects_v3/geometry.py

```diff
@@ -0,0 +1,97 @@
+from dataclasses import dataclass
+from specklepy.objects_v3.models.base import Base
+from specklepy.objects_v3.primitive import Interval
+from specklepy.objects_v3.interfaces import ICurve
+
+
+@dataclass(kw_only=True)
+class Point(Base, speckle_type="Objects.Geometry.Point"):
+    """
+    a 3-dimensional point
+    """
+    x: float
+    y: float
+    z: float
+    units: str
```

Comment:
there is already a units implementation in Base, we should port  that to an IHasUnits and inherit from that.